lib/hooks/discard/index.js

```diff
@@ -6,6 +6,7 @@ module.exports = settings => {
   return model => {
     const type = get(model, 'data.model');
     const action = get(model, 'data.action');
+    const changedBy = get(model, 'meta.user.profile.id');
```

Comment:
added this because the recall RA hook includes it, but it doesn't appear to be used anywhere other than the logs?